### PR TITLE
CASM-4072:  Improve IUF documentation for managed nodes rollout

### PR DESCRIPTION
**Pull request by jsollom-hpe**
The documentation now links to an explanation for configuring the Slurm workload manager to enable 'rolling updates' which the managed node rollout functionality uses. Further, it lists the command to Slurm to execute the reboot through BOS.

# Description

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/boot_orchestration/Rolling_Upgrades.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ For more information on staging sessions, see [Creating a Staged Session](./Stag
 1. The workload manager calls the `applystaged` endpoint for each node when it is ready.
 BOS then copies the information staged for these components into their desired state, and BOS starts to operate on the nodes and attempts to make their actual state match their new desired state.
 
+<a name="staged-sessions-with-slurm"></a>
```
Comment (deantroe) [inline operations/boot_orchestration/Rolling_Upgrades.md:28]: I think @haasken-hpe call this out in another PR, but these sorts of entries are not necessary (or even allowed by the repo checks?).  You can just reference the section by name in Markdown without adding anything special like this.

**Review by jsl-hpe — APPROVED** [review]

**Review by deantroe — CHANGES_REQUESTED** [review]
Just a few suggestions.

File: operations/boot_orchestration/Rolling_Upgrades.md (hunk 1 above)
Comment (jsollom-hpe) [reply operations/boot_orchestration/Rolling_Upgrades.md:28]: I had removed it from main and thought the cherry-pick to 1.4 had gotten it, but I was wrong.
Will remove.

File: operations/iuf/workflows/managed_rollout.md [hunk 2]
```diff
@@ -59,15 +59,28 @@ referred to as the "canary node" in the remainder of this section. Use `--limit-
 
 1. Verify the canary node booted successfully with the desired image and CFS configuration.
 
-1. Invoke `iuf run` with `-r` to execute the [`managed-nodes-rollout`](../stages/managed_nodes_rollout.md) stage on all nodes, rebooting the nodes in the default staged manner in conjunction with the workload manager. If an
-immediate reboot of compute nodes is desired instead, add `-mrs reboot` to the `iuf run` command.
+1. Invoke `iuf run` with `-r` to execute the [managed-nodes-rollout](../stages/managed_nodes_rollout.md) stage on all nodes. This will stage data to BOS and allow the workload manager to reboot the nodes when it is ready to do so. The workload managers
+must be configured to tell BOS to reboot nodes using this staged data. The following assumes Slurm is the workload manager.
+Follow the instructions in the section ['Using staged sessions with Slurm'](../../boot_orchestration/Rolling_Upgrades.md#Using-staged-sessions-with-slurm)
```
[the remark this answers is not in the document]
Comment (jsollom-hpe) [reply operations/iuf/workflows/managed_rollout.md:64]: Fixed.

File: operations/iuf/workflows/managed_rollout.md [hunk 3]
```diff
@@ -59,15 +59,28 @@ referred to as the "canary node" in the remainder of this section. Use `--limit-
...
+of the Rolling Upgrades using BOS. These instructions describe two parameters that must be set in the slurm.conf file. Return to these instructions after setting them.
```
[the remark this answers is not in the document]
Comment (jsollom-hpe) [reply operations/iuf/workflows/managed_rollout.md:65]: documentation-ed and linked.

[the remark this answers is not in the document]
Comment (jsollom-hpe) [reply operations/iuf/workflows/managed_rollout.md:65]: Ticked.

[the remark this answers is not in the document]
Comment (jsollom-hpe) [reply operations/iuf/workflows/managed_rollout.md]: I put backticks around slurm.conf, but that changes the formatting. Did you want the backticks escaped, or were you telling me you wanted the formatting the backticks introduces?

File: operations/iuf/workflows/managed_rollout.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jsollom-hpe) [reply operations/iuf/workflows/managed_rollout.md]: Changed.

File: operations/iuf/workflows/managed_rollout.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jsollom-hpe) [reply operations/iuf/workflows/managed_rollout.md]: I assumed you wanted them escaped.

[the remark this answers is not in the document]
Comment (deantroe) [reply operations/iuf/workflows/managed_rollout.md]: Sorry, the other way (where it changes the format).  it's my understand file names are supposed to be formatted this way, much like commands, prompts, etc.  I added a suggestion comment to make it clear.

File: operations/iuf/workflows/managed_rollout.md [hunk 4]
```diff
@@ -59,15 +59,43 @@ referred to as the "canary node" in the remainder of this section. Use `--limit-
...
     iuf -a "${ACTIVITY_NAME}" run -r managed-nodes-rollout
     ```
 
+   **NOTE:** If the `-mrs reboot` option is used, skip the following step.
+
+1. Tell the workload manager to reboot the compute nodes. This only works for compute nodes, and they must be specified explicitly. Using the keyword `ALL` to specify the nodes does not work presently.
+
+    - Use this command to list all of the compute nodes in the system using their Node Identities (NIDs). First, enter the SAT bash shell using `sat bash`.
+
+    (`ncn-m001#`)
```
Comment (deantroe) [inline operations/iuf/workflows/managed_rollout.md:81]:
```suggestion
    (`ncn-m001#`) sat bash
```

**Review by deantroe — APPROVED** [review]
Just 2 more comments.

File: operations/iuf/workflows/managed_rollout.md [hunk 5]
```diff
@@ -59,15 +59,43 @@ referred to as the "canary node" in the remainder of this section. Use `--limit-
...
+
+    ```bash
+    (`sat bash`)
```
[the remark this answers is not in the document]
Comment (jsollom-hpe) [reply operations/iuf/workflows/managed_rollout.md]: Fixed.

**Pull request merged**